### ISSUE TITLE
Accessibility improvements: add labels to edit button

### ISSUE THREAD

**Issue by Rathch** [b13/container#637]
Problem description
Container on individual pages can be edited in the ‘Page’ area. In the corresponding boxes for the container elements, there is an icon in the top right-hand corner that indicates the edit function. This icon has neither link text in the <a> element nor a programmatically determinable label. The function of the icon is therefore not accessible to screen reader users.

Recommended action
The icon must be provided with a meaningful, programmatically determinable label. This can be done by means of visible link text, an aria-label attribute or an assignment via aria-labelledby. The purpose of the element (e.g. ‘Edit content’) must be clearly recognisable and readable by assistive technologies.

internal reference: 5813

<img width="552" height="146" alt="Image" src="https://github.com/user-attachments/assets/6ae8a314-f9e5-423d-ab85-83716caf7ae2" />
